ui-tests/cypress/integration/content/templates.spec.js

```diff
@@ -0,0 +1,91 @@
+import HomePage from '../../support/pageObjects/HomePage';
+import { htmlElements } from '../../support/pageObjects/WebElement';
+import { generateRandomId, generateRandomNumericId } from '../../support/utils';
+
+const openContentTemplatesPage = () => {
+  cy.visit('/');
+  let currentPage = new HomePage();
+  currentPage     = currentPage.getMenu().getContent().open();
+  return currentPage.openTemplates();
+};
+
+const addContentTemplate = template => cy.contentTemplatesController().then(controller => controller.addContentTemplate(template));
+const deleteContentTemplate = id => cy.contentTemplatesController().then(controller => controller.deleteContentTemplate(id));
+
+describe('Content Templates', () => {
+
+  let currentPage;
+  let template = {
+    contentType: 'BNR',
+    contentTypeText: 'Banner',
+    contentShape: '<div>test</div>'
+  };
+
+  beforeEach(() => {
+    template.id = generateRandomNumericId();
+    template.descr = generateRandomId();
+
+    cy.kcLogin('admin').as('tokens');
+  });
+
+  afterEach(() => {
+    deleteContentTemplate(template.id);
```

Comment:
This is going to fail in '_Delete content template_' scenario; you need to add a check for each scenario to verify if there's a content template to be deleted

Reply:
It doesn't let the test fail but indeed it does produce an error in the log. Made the necessary changes. Thanks!